src/config/dynamic-theme-fixes.config

```diff
@@ -157,6 +157,19 @@ body > #root > #wrapper > div[class^='c'] > #main-content > div:first-child
 
 ================================
 
+*.vultr.com
+
+CSS
+.svg-banner-shape {
+    fill: var(--darkreader-neutral-background);
+}
+
+IGNORE INLINE STYLE
+svg
```

Comment:
These are too fague and will target all current SVG's and future one's.

Reply:
Alright, I'll expand them a bit them.

Reply:
Can you go through it again?